tutorials/conf.py

```diff
@@ -227,3 +223,16 @@
 
 # A list of files that should not be packed into the epub file.
 epub_exclude_files = ['search.html']
+
+# -- Options for myst_nb -----------------------------------------------------
+nb_execution_timeout = 120 # in seconds
+nb_execution_in_temp = False
```

Comment:
Just as a note: rn we can't build/exec on temporary dir because of the data directory -- basically the notebook will lose the relative path reference